src/visualization/types/Table/TableCell.test.ts

```diff
@@ -0,0 +1,108 @@
+import {asLink} from './TableCell'
+
+describe('TableCell.asLink', () => {
+  it('Properly handles various base data types', () => {
+    expect(asLink('a regular string')).toEqual('a regular string')
+    expect(asLink(NaN)).toEqual(NaN)
+    expect(asLink(true)).toEqual(true)
+    expect(asLink(42)).toEqual(42)
+    expect(asLink(undefined)).toEqual(undefined)
+    expect(asLink(null)).toEqual(null)
```

Comment:
can we get an `array`, `object`, and `json object` to test here

Reply:
Added per our conversation